### PR TITLE
Handle connection failures

### PR DESCRIPTION
**Pull request by dbrgn**
- What if a sensor has the wrong value?
- What if redis is down?

There should be proper error logging.

### CODE REVIEW

**Comment by dbrgn** [comment]
@rnestler please review.

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Some small shortcuts
 
+all:
+	cargo run
+
 docs:
```
Comment (rnestler) [inline Makefile]: Hmm... What's that?

File: spaceapi_server/src/sensors.rs [hunk 2]
```diff
@@ -19,10 +19,10 @@ impl SensorSpec {
...
         let datastore_lock = datastore_clone.lock().unwrap();
-        match datastore_lock.retrieve(&self.data_key) {
-            Ok(v) => Some(v),
-            Err(_) => None,
-        }
+        datastore_lock.retrieve(&self.data_key).map_err(|err| {
+            warn!("Could not retrieve key '{}' from datastore, omiting the sensor", &self.data_key);
+            err
+        }).ok()
     }
```
Comment (rnestler) [inline spaceapi_server/src/sensors.rs]: Maybe we should log the value of `err` as well?

File: Makefile (hunk 1 above)
Comment (dbrgn) [inline Makefile]: So you can type `make` to run cargo :)

File: spaceapi_server/src/sensors.rs (hunk 2 above)
Comment (dbrgn) [inline spaceapi_server/src/sensors.rs]: We could. Maybe as debug?

**Comment by dbrgn** [comment]
@rnestler can we merge this now?

**Comment by rnestler** [comment]
The Makefile change has nothing to do with the pull request but otherwise I think its ready to get merged

**Pull request merged**